app/notifications/receive_notifications.py

```diff
@@ -56,6 +56,10 @@ def receive_mmg_sms():
 def receive_firetext_sms():
     post_data = request.form
 
+    # This is a pre-implementation test code to validate the provider is sending through what they claim.
+    auth = request.authorization
+    current_app.logger.info("Inbound sms username: {}".format(auth.username))
```

Comment:
I think `request.authorization` can be `None` when the header is empty, so `auth.username` will raise an AttributeError if we don't wrap this with `if auth: ...`